src/uu/who/src/who.rs

```diff
@@ -264,10 +264,13 @@ pub fn uumain(args: impl uucore::Args) -> i32 {
             assumptions = false;
         }
 
-        if matches.is_present(options::RUNLEVEL) {
-            need_runlevel = true;
-            include_idle = true;
-            assumptions = false;
+        #[cfg(any(target_vendor = "apple", target_os = "linux", target_os = "android"))]
```

Comment:
This is also not supported on macOS, the manpage for BSD's `who` says:
`-r    Print the current runlevel.  This is meaningless on Mac OS X.`